tests/zenko_tests/node_tests/config.json

```diff
@@ -1,6 +1,6 @@
 {
     "reporterEnabled": "spec, mocha-junit-reporter",
     "mochaJunitReporterReporterOptions": {
-        "mochaFile": "junit-node-[hash].xml"
+        "mochaFile": "/artifacts/junit-node-[hash].xml"
```

Comment:
should be configurable, these tests are also run manually/locally....

Reply:
If they are ran, because of the way the tests works, by creating a pod inside the cluster and running the tests from within I assume they are run with the Dockerfile that is available, in which the volume `/artifacts/` has been created.

In this particular spot, I see no need of making this configurable and I don't even think those config would support it.

Reply:
The main issue we face for running the tests "locally" is zenko install, setup of locations/buckets/..., and all the required env variables: the test themself should be able to run outside of cluster. This should be addressed with the .devcontainer / codespace.

So we must not add to the debt by requiring the test to run in a pod or container...

Reply:
the patht to the junit report can be overriden by specifying another path in [the `MOCHA_FILE` env variable](https://github.com/michaelleeallen/mocha-junit-reporter/blob/master/README.md#usage)